src/js/pensions/config/form.js

```diff
@@ -755,6 +750,10 @@ const formConfig = {
                 reasonForSeparation: {
                   'ui:title': 'Why did the marriage end?',
                   'ui:widget': 'radio'
+                },
+                dateOfSeparation: currentOrPastDateUI('Date marriage ended'),
+                locationOfSeparation: {
+                  'ui:title': 'Place marriage ended',
```

Comment:
The paper form doesn't make any distinction and based on the walkthrough, I think these fields were expected for widows.